lib/services/auth-service.js

```diff
@@ -37,23 +39,12 @@ function authServiceFactory(
 
     AuthService.prototype.init = function(){
         var self = this;
-        this.strategies = [];
 
         self.algorithm = 'HS256';
         self.secretOrKey = '';
         self.ignoreTokenExpiration = false;
         self.encoder = 'base64';
 
-        self.hashConfig = {
```

Comment:
is removing this reducing the strength of the hashing, or was this the default and simply redundant?